### PR TITLE
feat : created a landing page of notes-app

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: src/pages/Nav.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+
+        <Routes>
+        {/* <Route path="/trash"  element={ <Trash /> } /> */}
+        <Route path="/home" element={<Home />} />
+        <Route path="/docs" element={<Docs />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/label" element={<Label />} />
```
Comment (saifudinbadani) [inline src/pages/Nav.js:60]: you can give this paths in index.js as well, easy to fetch and edit.

**Review by saifudinbadani — APPROVED** [review]
Rest looks good to me.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
+      {/* <CreateNote /> */}
```
Comment (MiheerTamkhane) [inline src/App.js:8]: Remove comments if not needed

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../css/home.css";
+const Home = () => {
+  return (
+    <>
+      {/* <h1> Home Page</h1> */}
```
Comment (MiheerTamkhane) [inline src/pages/Home.js:7]: this one also

File: src/pages/Nav.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import "../css/nav.css"
+
+import { Route,Routes,Link } from "react-router-dom";
+import Home from "./Home";
+import Docs from "./Docs";
+import Label from "./Label";
+import Login from "./Login";
+import Notes from "./Notes";
+import Remainder from "./Remainder";
+import Trash from "./Trash"
```
Comment (MiheerTamkhane) [inline src/pages/Nav.js:11]: Barrel export can be good to use here

File: src/pages/Nav.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        <div className="navbar navbar-expand-lg navbar-light bg-warning nav-left">
+          <Link className="nav-link " to="/home">
+            <i className="fa fa-star star-logo" aria-hidden="true"></i>
+            Star Notes
+          </Link>
+        </div>
+
+        <div className="navbar navbar-expand-lg navbar-light bg-warning justify-content-end nav-right">
+          <button  className="nav-link btn">
+            {" "}
```
Comment (MiheerTamkhane) [inline src/pages/Nav.js:25]: remove spaces also

**Review by MiheerTamkhane — APPROVED** [review]
Code is good and readable 
suggestion : I see you have imported link.css in too many files you can only import in index.css and can use everywhere.
alos you can remove React import if not used.

File: src/pages/Remainder.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <>
+      <div className="wrapper">
+        <div className="slider">
+          <SubLink />
+        </div>
+
+        <div className="slider-content">
+          <h1> Remainder PAGE</h1>
+        </div>
```
Comment (BelideNikhil) [inline src/pages/Remainder.js:15]: text inconsistency can be avoided

**Pull request merged**